lib/client/bgppeer_e2e_test.go

```diff
@@ -186,109 +189,3 @@ var _ = testutils.E2eDatastoreDescribe("BGPPeer tests", testutils.DatastoreEtcdV
 		})
 	})
 })
-
-// Perform CRUD operations on Global BGP Peer Resources
-var _ = testutils.E2eDatastoreDescribe("Global BGPPeer tests", testutils.DatastoreAll, func(config api.CalicoAPIConfig) {
```

Comment:
The guys ^^ should be sufficient.  Previously I couldn't flag the guys ^^ to be tested against KDD because of the lack of Per-Node BGP support.  Now that I can, I can get rid of these guys down below which were basically a copy of the guys above (but without any Per-node BGP Peers).